### PR TITLE
Feature library merge request

### PR DESCRIPTION
**Pull request by moinkhanif**
Some of the features on the projects include:

- Add new books to the library
- Delete books from the library
- Check read status of the given books

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    "supports-color": "^5.5.0",
+    "tmp": "^0.0.33",
+    "through": "^2.3.8",
+    "type-check": "^0.3.2",
+    "tslib": "^1.13.0",
+    "uri-js": "^4.2.2",
+    "type-fest": "^0.8.1",
+    "cli-cursor": "^3.1.0",
+    "function-bind": "^1.1.1",
+    "argparse": "^1.0.10"
```
Comment (Torres-ssf) [inline package.json]: I couldn't help noticing that your package json file is has a lot installed dependencies that are not being used by your app. Can I ask why those dependencies were added :thinking: ?

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi team :hand: 
Great job in this project :clap: 
You are almost done with it, we just need to make some changes before moving on.
 
- [x] Right now it's possible to create a new book without providing any information. You can change that by applying the `required` HTML attribute to all the input tags (Title, Author, Pages). This way the form will complain if someone tries to create a new blank book.

- [x] The read toggle button it's not toggling the book read status.

- [x] The section Features from your README file has an "empty item dot" that can be removed. 

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

File: package.json (hunk 1 above)
Comment (moinkhanif) [reply package.json]: Hey @Torres-ssf, we initially tried to install bootstrap toggle package and later removed it. I just noticed these. I have updated the same. Thank you for pointing it out. 
Good day 👍

**Review by Redvanisation — APPROVED** [review]
# Approved ✔️ 

### Hello team 🌟 

### _Nice work on this project, it seems that you have met the requirements. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

**Pull request merged**